src/acceptance-test/java/de/zalando/aruha/nakadi/repository/db/EventTypeDbRepositoryTest.java

```diff
@@ -146,6 +146,20 @@ public void whenListExistingEventTypesAreListed() throws NakadiException, Duplic
         assertThat(eventTypes, hasSize(2));
     }
 
+    @Test
+    public void whenRemoveThenDeleteFromDatabase() throws Exception {
+        EventType eventType = buildEventType();
+        String insertSQL = "INSERT INTO zn_data.event_type (et_name, et_event_type_object) VALUES (?, to_json(?::json))";
```

Comment:
It is considered to be a bad practice to use the system/class you test for setup of a test.

In two words, if you have a system A that can do get(X) and set(X) - you should not use set(X) for setup of test for get(X) (and vise versa). Because if a test of system A you can't yet trust system A. And also if you have bug in both set(X) and get(X) you test can be false positive.